style.css

```diff
@@ -0,0 +1,318 @@
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  color: #172b4d;
+}
+
+h3 {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  color: #344563;
+}
+
+h3 span {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  color: #7a869a;
+}
+
+h5 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  color: #172b4d;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 30px auto;
+}
+
+.canopy-card {
+  background: #f7f7f9;
+  padding: 20px 0;
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  grid-auto-rows: auto;
+  grid-gap: 1rem;
+}
+
+.snapshoot-portofolio {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.canopy-card-section {
+  display: flex;
+  background: #fff;
+  height: 529px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.canopy-second-card-section {
+  background: #fff;
+  height: 563px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.canopy-third-card-section {
+  background: #fff;
+  height: 519px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.canopy-fourth-card-section {
+  background: #fff;
+  height: 563px;
+  width: 327px;
+  left: 24px;
+  top: 754px;
+  border-radius: 16px;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  margin: 15px auto;
+}
+
+.project-card {
+  width: 100%;
+  margin: -5px auto;
+}
+
+.project-card p {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  color: #344563;
+}
+
+.project-card li {
+  display: inline-block;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03;
+  color: #6070ff;
+  background: #ebebff;
+  margin-right: 5px;
+  padding: 7px;
+  list-style: none;
+  border-radius: 8px;
+  transition-duration: 0.4s;
+}
+
+.project-card li:hover {
+  background-color: gray;
+}
+
+.project-card ul {
+  padding: 0;
+  width: 300px;
+}
+
+button {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  margin-top: 30px;
+  padding-top: 12px;
+}
+
+.project-card button {
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03;
+  color: #396df2;
+  border: 1px solid #396df2;
+  padding: 7px 16px;
+  margin: 16px auto;
+  border-radius: 8px;
+  background: #fff;
+  cursor: pointer;
+  flex-wrap: wrap;
+  transition-duration: 0.5s;
+}
+
+.project-card button:hover {
+  background-color: blue;
+}
```

Comment:
you have done a good job implementing requested changes by previous reviewer but make sure to align your design with figma design the color you used is for  press event check this image :

![hover button](https://user-images.githubusercontent.com/59707859/219692776-ae92dafc-36fe-40ff-b1f3-63a438ef8ffe.PNG)

Reply:
corrected